src/spyglass/ripple/v1/ripple.py

```diff
@@ -239,6 +239,7 @@ def make(self, key):
 
     def fetch1_dataframe(self) -> pd.DataFrame:
         """Convenience function for returning the marks in a readable format"""
+        self.ensure_single_entry()
```

Comment:
I prefer the following notation for indicating the function does not return anything of use
```suggestion
        _ = self.ensure_single_entry()
```
but I don't mind merging as-is